napari/_tests/test_viewer.py

```diff
@@ -135,5 +135,58 @@ def test_changing_theme(viewer_factory):
     viewer.theme = 'light'
     assert viewer.palette['folder'] == 'light'
 
+
+    # Close the viewer
+    viewer.window.close()
+
     with pytest.raises(ValueError):
         viewer.theme = 'nonexistent_theme'
+        
+        
+def test_roll_traspose_update(qtbot):
```

Comment:
Similarly here I think you can use `viewer_factory` instead of `qtbot` - see [writing tests](https://github.com/napari/napari/blob/master/docs/developers/CONTRIBUTING.md#writing-tests) in our contributing guide. Right @tlambert03?

Reply:
You will then be able to drop the `viewer.window.close` bellow too

Reply:
yep